Calculator.py

```diff
@@ -0,0 +1,69 @@
+'''
+Solution:
+1.  If a digit is occurred, add the value to the current value.
+2.  If an operator is found, push the current value to the stack and refresh the current value.
+3.  At the end, add all values in the stack.
+
+Time Complexity:    O(n)    | Space Complexity: O(n)
+
+--- Passed all testcases successfully on Leetcode
+'''
+
+
+class Calculator:
+    def calculate(self, s: str) -> int:
+        
+        #   edge case check
+        if (s == None or len(s) == 0):
+            return 0
+        
+        #   initializations
+        digits = '0123456789'
+        operators = '+-*/'
+        
+        stack = []
+        cursor = 0
+        sign = '+'
+        value = 0
+        
+        #   iterate until the cursor doesn't cross the length of the dtring
+        while (cursor < len(s)):
+            
+            #   if value occurred
+            if (s[cursor] in digits):
```

Comment:
The traversal on digits string will be length of digits string. Use a set here for O(1)